src/pages/settings/Subscription/PaymentCard/index.tsx

```diff
@@ -10,53 +10,73 @@ import Section, {CARD_LAYOUT} from '@components/Section';
 import Text from '@components/Text';
 import TextLink from '@components/TextLink';
 import useLocalize from '@hooks/useLocalize';
+import usePreferredCurrency from '@hooks/usePreferredCurrency';
+import useSubscriptionPlan from '@hooks/useSubscriptionPlan';
+import useSubscriptionPrice from '@hooks/useSubscriptionPrice';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as CardUtils from '@libs/CardUtils';
-import Navigation from '@navigation/Navigation';
+import {convertToShortDisplayString} from '@libs/CurrencyUtils';
 import * as PaymentMethods from '@userActions/PaymentMethods';
 import CONST from '@src/CONST';
-import type ONYXKEYS from '@src/ONYXKEYS';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
 
 function AddPaymentCard() {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
+    const [privateSubscription] = useOnyx(ONYXKEYS.NVP_PRIVATE_SUBSCRIPTION);
+    const [fundList] = useOnyx(ONYXKEYS.FUND_LIST);
+    const [shouldShowPaymentCardForm, setShouldShowPaymentCardForm] = useState(false);
+
+    const subscriptionPlan = useSubscriptionPlan();
+    const subscriptionPrice = useSubscriptionPrice();
+    const preferredCurrency = usePreferredCurrency();
+
+    const isCollect = subscriptionPlan === CONST.POLICY.TYPE.TEAM;
+    const isAnnual = privateSubscription?.type === CONST.SUBSCRIPTION.TYPE.ANNUAL;
+    const defaultCard = useMemo(() => Object.values(fundList ?? {}).find((card) => card.isDefault), [fundList]);
```

Comment:
Should this be this?
```suggestion
    const defaultCard = useMemo(() => Object.values(fundList ?? {}).find((card) => card.accountData?.additionalData?.isBillingCard), [fundList]);
```

I'm updating how we access this in `CardSection.tsx`, not sure if you were copying from there. See [PR](https://github.com/Expensify/App/pull/44361).

Reply:
Oh did not see that. Sure will push that change